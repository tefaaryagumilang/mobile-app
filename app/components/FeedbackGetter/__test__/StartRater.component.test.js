import React from 'react';
import renderer from 'react-test-renderer';
import StarRater from '../StarRater.component';

describe('StarRater component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<StarRater />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
