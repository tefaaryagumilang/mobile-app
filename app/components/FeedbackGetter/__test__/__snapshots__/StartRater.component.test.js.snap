// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarRater component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "center",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#C1000E",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 5,
                "padding": 10,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#C1000E",
                "borderRadius": 28,
                "borderWidth": 2,
                "marginLeft": 2,
                "padding": 2,
                "shadowColor": "#C1000E",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            0
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#C1000E",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 5,
                "padding": 10,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#C1000E",
                "borderRadius": 28,
                "borderWidth": 2,
                "marginLeft": 2,
                "padding": 2,
                "shadowColor": "#C1000E",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            1
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#C1000E",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 5,
                "padding": 10,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#C1000E",
                "borderRadius": 28,
                "borderWidth": 2,
                "marginLeft": 2,
                "padding": 2,
                "shadowColor": "#C1000E",
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            2
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#D46B07",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D46B07",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            3
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#D46B07",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D46B07",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            4
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#D46B07",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#D46B07",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            5
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#E6D000",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#E6D000",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            6
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#E6D000",
                "borderRadius": 13.5,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#E6D000",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            7
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#AEC317",
                "borderRadius": 20,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#AEC317",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            8
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#AEC317",
                "borderRadius": 20,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#AEC317",
                "borderRadius": 28,
                "borderWidth": 2,
                "fontSize": 25,
                "marginLeft": 2,
                "padding": 2,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "color": "#000000",
                "fontSize": 11,
                "padding": 1,
              }
            }
          >
            9
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "borderColor": "#23A251",
                "borderRadius": 20,
                "borderWidth": 2,
                "height": 27,
                "marginLeft": 4,
                "padding": 4,
                "width": 27,
              },
              Object {
                "backgroundColor": "#FFFFFF",
                "borderColor": "#23A251",
                "borderRadius": 15,
                "borderWidth": 2,
                "fontSize": 18,
                "marginLeft": 2,
                "marginRight": 2,
                "padding": 0.8,
              },
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#000000",
                "fontSize": 10,
                "paddingBottom": 3,
                "paddingTop": 3,
              }
            }
          >
            10
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
